### ISSUE TITLE
Incompatible with FTB Utilities

### ISSUE THREAD

**Issue by hideki2k02** [dzwdz/chat_heads#113]
Hello, I was looking through the Closed Issues and apparently it should be compatible with /nickname commands, however that does not seem to be the case with FTB Utilities. Could you take a look into that?
![image](https://github.com/user-attachments/assets/ef50d1b4-172b-490a-af28-ad59ce2aab81)

**Comment by Fourmisain** [comment]
Isn't FTB Utilities for 1.12.2? Ah, you mean FTB Essentials.

> I was looking through the Closed Issues and apparently it should be compatible with /nickname commands

Yesn't, Chat Heads can only do so much.
Your server is sending system messages in place of player messages, which do not have player identifying information (UUIDs). Chat Heads can thus only try matching the message to a player via the known names, which are profile names and tab display names.

In this case FTB Essentials [modifies the player entity's display name](https://github.com/FTBTeam/FTB-Library/blob/bd349d336776b231e3a3da70d9fb4a9208ce0d1f/fabric/src/main/java/dev/ftb/mods/ftblibrary/core/mixin/fabric/PlayerMixin.java#L18-L27), which affects chat, but not the tab display name:

![names](https://github.com/user-attachments/assets/f430b8a7-b941-4945-80fe-fc03e99d8cbe)

In other words, I don't think there's anything Chat Heads can do in this case.

There is however a "Name aliases" feature in the config where you can match your nickname to either your profile or tab display name, like `Duck -> profile_name42`:

![alias](https://github.com/user-attachments/assets/7fe05e65-c656-41fe-a043-c000a38536c6)

**Comment by hideki2k02** [comment]
What a shame :(
It might be worth it to leave the issue open or leave a incompatibility notice somewhere. What I did to keep ChatHeads was disabling /nick from FTB Essentials (serverconfig folder) and switching to another mod (on my case Styled Nicknames).
